config/settings.go

```diff
@@ -124,11 +126,13 @@ func (s *Settings) LoadFromEnv() error {
 	if s.LastSnapshotName == "" {
 		s.LastSnapshotName = "cg-last-snapshot"
 	}
-
-	// variables for connecting to cf api
-	// s.CfUser = os.Getenv("CF_USERNAME")
-	// s.CfPass = os.Getenv("CF_PASSWORD")
-	// s.CfApiUrl = os.Getenv("CF_API_URL")
-
+	s.MaxBackupRetention, _ = strconv.ParseInt(os.Getenv("MAX_BACKUP_RETENTION"), 10, 64)
+	if s.MaxBackupRetention == 0 {
+		s.MaxBackupRetention = 35
+	}
+	s.MinBackupRetention, _ = strconv.ParseInt(os.Getenv("MIN_BACKUP_RETENTION"), 10, 64)
+	if s.MinBackupRetention == 0 {
+		s.MinBackupRetention = 1
```

Comment:
It's awesome to have this flexibility!  However, based on the NIST 800-53 controls and FedRAMP guidelines we may need to default to a 14 day minimum.

I _think_ CP-9 is the control that we adhere to ourselves and offer our customers (especially per the [control implementation summary](https://cloud.gov/resources/cloud.gov-CIS-Worksheet.xlsx) and our SSP), and if we allow them to adjust below that it might make things more complicated.

```suggestion
		s.MinBackupRetention = 14
```

Reply:
I wondered about this myself, but seems reasonable to me to limit it to the 14 day as min. Thanks!